packages/theme-chalk/src/input.scss

```
@@ -237,7 +237,9 @@
     );
     cursor: text;
     transition: getCssVar('transition-box-shadow');
-    transform: translate3d(0, 0, 0);
+    // Keep the 3D transform to avoid Chromium inset box-shadow noise.
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)